README.md

```diff
@@ -103,10 +103,9 @@ further customize the QM bluechi agent by adding content to the
 ## QM Sub-packages
 
 The qm project is designed to provide a flexible and modular environment for managing
-Quality Management (QM) software in containerized environments. One of the key features
-of the qm package is its support for sub-package(s), such as the qm-dropin sub-packages.
-These sub-packages are not enabled by default and are optional. However,  allow users
-to easily extend or customize their QM environment by adding specific configurations,
+Quality Management (QM) software in containerized environments.
+Sub-packages are not enabled by default and are optional. However, allow users
+to extend or customize their QM environment by adding specific configurations,
```

Comment:
Please add this comment some where
There should be correlation between subpackage version and qm version
In case rpm should be generated for qm-version-distro?
Make sure to update the VERSION file to the QM version.